static/scss/_variables.scss

```diff
@@ -16,3 +16,10 @@ $search-filter: #D2E4F9;
 // other
 $block-padding: 20px;
 $page-padding: 40px;
+$progress-widget-bg-color: #ddd;
+$progress-widget-fg-color: #2ecc40;
```

Comment:
No it is different, because $mm-brand-bg is the color i am using in background.
These are different colors so that they can appear in from of progress widget background